packages/hardhat-zksync-deploy/src/zksync-deploy-plugin-error.ts

```diff
@@ -0,0 +1,8 @@
+import { HardhatPluginError } from 'hardhat/plugins';
+import { PLUGIN_NAME } from './constants';
+
+export class ZkSyncDeployPluginError extends HardhatPluginError {
+    constructor(message: string, parentError?: Error) {
+        super(PLUGIN_NAME, message, parentError);
+    }
+}
```

Comment:
Why can't we just put this in utils.ts? What's the reason for this monstrous filename?

Reply:
In my opinion utils.ts is not a perfect place for defining errors. My preference is to have a separate file for this, although I do agree that the file name is a bit long, so maybe I can just rename it to 'errors.ts' and keep it as a place for defining all errors in the future. Is this ok for you?